src/schoology-extractor/edfi_schoology_extractor/helpers/csv_reader.py

```diff
@@ -1,24 +1,24 @@
-# SPDX-License-Identifier: Apache-2.0

```

Comment:
I wonder if we should just go ahead and add linefeed style to the editorconfig file to avoid having some files with CRLF and some with LF. Hard to tell if any actual changes occurred here. I can see on my local side that the file was previously in CRLF and is now LF - which I'm happy about.

What do you think about adding `end_of_line = lf` to the editorconfig file? @lalrg any concern?

Reply:
Sounds good to me

Reply:
Sounds good

Reply:
I'll do that in my pull request and will accept & merge this one.